README.md

```diff
@@ -57,17 +57,17 @@ only supports gRPC as its communications protocol.
 | System property                  | Environment variable             | Purpose                                                              |
 |----------------------------------|----------------------------------|----------------------------------------------------------------------|
 | ota.exporter=jaeger              | OTA_EXPORTER=jaeger              | To select Jaeger exporter                                            |
-| ota.exporter.jaeger.endpoint     | OTA_EXPORTER_JAEGER_ENDPOINT     | The Jaeger endpoint to connect to. Currently only gRPC is supported. |
-| ota.exporter.jaeger.service.name | OTA_EXPORTER_JAEGER_SERVICE_NAME | The service name of this JVM instance                                |
+| JAEGER_ENDPOINT                  | JAEGER_ENDPOINT                  | The Jaeger endpoint to connect to. Currently only gRPC is supported. |
```

Comment:
Wrong format of System property name

Reply:
Yes and no 😄. It's an issue upstream. Should get fixed by https://github.com/open-telemetry/opentelemetry-java/issues/1408.

Reply:
yes, but it comes from `otel-java`: https://github.com/open-telemetry/opentelemetry-java/blob/master/exporters/jaeger/src/main/java/io/opentelemetry/exporters/jaeger/JaegerGrpcSpanExporter.java#L41

Reply:
The **environment** var `JAEGER_SERVICE_NAME` must be `OTEL_JAEGER_SERVICE_NAME` to work for me in 0.7.0-SNAPSHOT (current master head).
I'm a little bit confused... what is really expected? :smile_cat:

Reply:
it makes sense to have documented that in otel-java and here add a link to it, although link also can be broken in next release...

Reply:
hey @flecno! does this PR answer your question: https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/830

Reply:
Yes, thank you!